app/services/channels/approve_channel_transfer.rb

```diff
@@ -17,8 +17,6 @@ def perform
         original_owner_name = @channel.publisher.name
         channel_name = @channel.publication_title
 
-        contested_by.verified = true
```

Comment:
I think we still need this, when a user transfers a channel it isn't marked as verified. We'd end up with scenarios where Youtube/Twitch/Twitter channels are transferred but not verified.